app/components/layout/nav.tsx

```diff
@@ -117,9 +141,9 @@ export function Nav() {
                 })}
               </div>
               <div className="flex items-center justify-center p-4">
-                <button className="text-md h-10 w-[152px] cursor-not-allowed rounded-full bg-cyan-500 font-bold text-white">
-                  Sign Up <span className="text-xs">(SOON)</span>
-                </button>
+                <Button className="bg-cyan-500 text-xs h-10 hover:bg-cyan-500 cursor-not-allowed">
```

Comment:
do not override the color, because Button component is using primary color, which is the same color as Figma design